pandas/tests/test_strings.py

```diff
@@ -538,13 +538,6 @@ def test_extract(self):
         exp = DataFrame([[u('BAD__'), u('BAD')], er, er])
         tm.assert_frame_equal(result, exp)
 
-        # GH9980
-        # Index only works with one regex group since
-        # multi-group would expand to a frame
-        idx = Index(['A1', 'A2', 'A3', 'A4', 'B5'])
```

Comment:
do you have a test with an Index with 1 capture group (and more than one)? (which should now work)